plugins/slipping-away-users/frontend/public/templates/SlippingAwayUsersPage.html

```diff
@@ -0,0 +1,79 @@
+<script type="text/x-template" id="slipping-away-users-page">
+    <div v-bind:class="[componentId]">
+        <cly-header>
+            <template v-slot:header-left>
+                <h2> {{i18n('slipping-away-users.title')}} 
+                </h2>
+                <span class="el-icon-info" style="margin-left:10px" v-tooltip.top-center="description">
+                </span>
+            </template>
+        </cly-header>
+        <cly-main>
+            <slipping-away-users-filter> </slipping-away-users-filter>
+            <slipping-away-users-bar-chart> </slipping-away-users-bar-chart>
+            <slipping-away-users-table> </slipping-away-users-table>
+        </cly-main>
+    </div>
+</script>
+
+<!-- NOTE-LA: <slipping-away-users-filter> is a temporary created component until query filter with modal becomes available-->
+<script type="text/x-template" id="slipping-away-users-filter">
+    <cly-section style="min-height:150px">
+        <div class="bu-columns bu-is-gapless bu-is-mobile bu-is-centered" style="padding-top:1%">
+            <div class="bu-column bu-is-12-tablet bu-is-9-desktop bu-is-7-widescreen bu-is-6-fullhd">
+                <cly-qb-segmentation
```

Comment:
In some cases (e.g. when drill is not active) this component wouldn't be found. We need to handle those cases.

Reply:
But anyway it is a temporary one right now.